test/README.md

```diff
@@ -0,0 +1,55 @@
+## How do NTH end-to-end tests work?
+
+
+#### TL;DR
+Instead of expected/actual *values*, the tests evaluate expected/actual **states**. NTH's functionality is evaluated by triggering various interruption events on the cluster (spot interrupts, maintenance events) and ensuring the nodes and pods in the cluster are in their proper state (cordoned, tainted, evicted, etc.). The tests fail if the nodes/pods don't transition into expected states suggesting an issue in NTH logic or cluster setup.
+
+
+#### Summary
+This doc details how the end-to-end (e2e) tests work for aws-node-termination-handler (NTH) at a high-level. These tests are no different from normal integration tests in that they capture the functionality of NTH. However, the *assert-actual-equals-expected* pattern is not as explicit as in other e2e tests which can cause some confusions. We hope to bring clarity with the content below.
+
+
+#### Starting Tests
```

Comment:
can you add an example of running the drivers directly and using the preserve flag?  I think most of the people reading this README will be devs working on a PR and the preserve example would be extremely valuable for them.

Reply:
added **Using Test Drivers** `Starting Tests` section